man/systemd.service.xml

```diff
@@ -420,6 +420,21 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ExecStartEx=</varname></term>
+        <term><varname>ExecStartPreEx=</varname></term>
+        <term><varname>ExecStartPostEx=</varname></term>
```

Comment:
oh, let's not mention this in the man page. i.e. while normally dbus properties and their counterparts in unit files are very closely related, there are plenty exceptions, and in this case there should be one too.

i.e. for the unit file setting ExecStart= there should be two dbus props: ExecStart= and ExecStartEx= (the former for compat the latter would be current). But there would not be a unit file setting ExecStartEx=, since the normal ExecStart= could already do that.

Other examples where the dbus props and unit file config settings don't match 1:1 are the Sec/USec cases for example: in unit files we expose all time values in properties suffixed with Sec, to indicate their default unit is second (example: TimeoutStartSec=), but on the bus they are exposed as TimeoutStartUSec=, since they are actually internally maintained in µs accuracy.